### PR TITLE
Lwrp support

### PR DESCRIPTION
**Pull request by jimmy-jam**
Update LWRP support branch so that it works with latest LWRP (4.7.0-preview) on 2018.3

### CODE REVIEW

File: Assets/LWRPSupport/LWRPBeforeCameraRender.cs [hunk 1]
```diff
@@ -1,27 +1,30 @@
 ﻿using UnityEngine.Experimental.Rendering;
-using UnityEngine.Rendering;
 using UnityEngine.Experimental.Rendering.LightweightPipeline;
+using UnityEngine.Experimental.Rendering.LWRP;
 
-namespace UnityEngine.XR.ARFoundation
+namespace UnityEngine.XR.ARFoundation.LWRPSupport
```
Comment (tdmowrer) [inline Assets/LWRPSupport/LWRPBeforeCameraRender.cs]: Do we need to introduce a new nested namespace here?

File: Assets/LWRPSupport/LWRPBeforeCameraRender.cs [hunk 2]
```diff
@@ -1,27 +1,30 @@
...
-			if (blitMaterial == null) 
-				return;
-			
-			var cmd = CommandBufferPool.Get(k_ARBlitTag);
-			
-			cmd.Blit(null, BuiltinRenderTextureType.CameraTarget, blitMaterial);
-
-			context.ExecuteCommandBuffer(cmd);
-			CommandBufferPool.Release(cmd);
+			LWRPBackgroundRenderPass lwrpBackgroundRenderPass = new LWRPBackgroundRenderPass();
```
Comment (tdmowrer) [inline Assets/LWRPSupport/LWRPBeforeCameraRender.cs]:
```suggestion
			var lwrpBackgroundRenderPass = new LWRPBackgroundRenderPass();
```

File: Packages/manifest.json [hunk 3]
```diff
@@ -1,17 +1,21 @@
...
     "com.unity.modules.particlesystem": "1.0.0",
     "com.unity.modules.physics": "1.0.0",
     "com.unity.modules.ui": "1.0.0",
+    "com.unity.modules.unityanalytics": "1.0.0",
+    "com.unity.modules.unitywebrequest": "1.0.0",
     "com.unity.modules.vr": "1.0.0",
     "com.unity.modules.xr": "1.0.0"
-  }
+  },
+  "registry": "https://staging-packages.unity.com"
```
Comment (tdmowrer) [inline Packages/manifest.json]: Remember to remove this before merging to the public repo.

File: Assets/LWRPSupport/LWRPBeforeCameraRender.cs (hunk 1 above)
Comment (jimmy-jam) [reply Assets/LWRPSupport/LWRPBeforeCameraRender.cs]: This is not new - the rest of the scripts in this folder is in this namespace already.

File: Packages/manifest.json (hunk 3 above)
Comment (jimmy-jam) [reply Packages/manifest.json]: LWRP 4.7.0 is still on staging - guess we can wait until it's pushed to prod?

Comment (tdmowrer) [reply Packages/manifest.json]: Yeah I think we should wait. Do we know when that will be?

File: Assets/LWRPSupport/LWRPBeforeCameraRender.cs (hunk 1 above)
Comment (tdmowrer) [reply Assets/LWRPSupport/LWRPBeforeCameraRender.cs]: Okay, didn't notice that. We should be consistent, but I still think a nested namespace here is odd, especially when all the types are already prefixed with `LWRP`. And `FeatureXSupport` feels like an unusual namespace name. What about removing the extra namespace from the other scripts?

**Comment by Dianna3D** [comment]
QA confirmed iOS and Android both compile and build without issue, approved.

**Comment by Dianna3D** [comment]
Clicked wrong button on Mobile.

**Pull request merged**